robot-server/robot_server/runs/run_store.py

```diff
@@ -32,26 +35,58 @@ def __init__(self, run_id: str) -> None:
 class RunStore:
     """Methods for storing and retrieving run resources."""
 
-    def __init__(self) -> None:
-        """Initialize a RunStore and its in-memory storage."""
-        self._runs_by_id: Dict[str, RunResource] = {}
+    def __init__(self, sql_engine: sqlalchemy.engine.Engine) -> None:
+        """Initialize a RunStore with sql engine."""
+        self._sql_engine = sql_engine
+        self._active_run = ""
```

Comment:
When no run is active, let's designate that with `None` instead of an empty string.

```suggestion
        self._active_run: Optional[str] = None
```